src/Unit/Velocity/MilePerHour.php

```diff
@@ -0,0 +1,14 @@
+<?php
+namespace PhpUnitConversion\Unit\Velocity;
+
+use PhpUnitConversion\System\Imperial;
+use PhpUnitConversion\System\USC;
+use PhpUnitConversion\Unit\Velocity;
+
+class MilePerHour extends Velocity implements Imperial, USC
```

Comment:
According to wikipedia the official unit is `Miles per hour`? (https://en.wikipedia.org/wiki/Miles_per_hour)

Strange as it seems its Kilometer per hour (singular) but Miles per hour (plural)

Reply:
Thinking about it a bit more I think its beter to keep this as MilePerHour but add a second class for MilesPerHour which only extends MilePerHour. That way we have the class name which is a combination of two other unit classes but also the official unit

Reply:
Done.